addons/delivery/models/sale_order.py

```diff
@@ -159,6 +159,9 @@ class SaleOrderLine(models.Model):
     def _is_not_sellable_line(self):
         return self.is_delivery or super(SaleOrderLine, self)._is_not_sellable_line()
 
+    def _is_eligible_for_total(self):
+        return not self.is_delivery or super(SaleOrderLine, self)._is_eligible_for_total()
```

Comment:
```suggestion
        return not self.is_delivery or super()._is_eligible_for_total()
```

Reply:
In fact, it should be, we use the propagate logical operations:

```suggestion
        return not self.is_delivery and super()._is_eligible_for_total()
```